utils/helpers/posts/getMumblePosts.ts

```diff
@@ -11,10 +11,14 @@ export const getMumblePosts = async ({
   offset = 0,
   limit = config.feed.defaultAmount,
   tags,
+  creators,
+  likedBy,
 }: {
   offset?: number;
```

Comment:
it probably make sense to extract those types to an IProps here

Reply:
I will improve that part in the coming recommendations PR.